src/leiningen/i18n/bin/update-pot.sh

```diff
@@ -2,8 +2,10 @@
 
 set -eo pipefail
 
+POT_NAME=""
```

Comment:
The commit message for when the POT changes are checked in (on line 66) should also be updated to use this here new POT_NAME variable.